x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/preconfigured_connector_flyout.tsx

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { useCallback } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiTitle,
+  EuiFlyoutHeader,
+  EuiFlyout,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiButtonEmpty,
+  EuiBetaBadge,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { EuiText } from '@elastic/eui';
+import { EuiLink } from '@elastic/eui';
+import { ActionConnectorTableItem } from '../../../types';
+import { useActionsConnectorsContext } from '../../context/actions_connectors_context';
+import { PLUGIN } from '../../constants/plugin';
+
+interface Props {
+  initialConnector: ActionConnectorTableItem;
+  flyoutVisible: boolean;
+  setFlyoutVisibility: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+export const PreconfiguredConnectorFlyout = ({
+  initialConnector,
+  flyoutVisible,
+  setFlyoutVisibility,
+}: Props) => {
+  const { actionTypeRegistry } = useActionsConnectorsContext();
+  const closeFlyout = useCallback(() => setFlyoutVisibility(false), [setFlyoutVisibility]);
+  if (!flyoutVisible) {
+    return null;
+  }
+
+  const actionTypeModel = actionTypeRegistry.get(initialConnector.actionTypeId);
+
+  return (
+    <EuiFlyout onClose={closeFlyout} aria-labelledby="flyoutActionAddTitle" size="m">
+      <EuiFlyoutHeader hasBorder>
+        <EuiFlexGroup gutterSize="s" alignItems="center">
+          {actionTypeModel ? (
+            <EuiFlexItem grow={false}>
+              <EuiIcon type={actionTypeModel.iconClass} size="m" />
+            </EuiFlexItem>
+          ) : null}
+          <EuiFlexItem>
+            <EuiTitle size="s">
+              <h3 id="flyoutTitle">
+                <FormattedMessage
+                  defaultMessage="{connectorName}"
+                  id="xpack.triggersActionsUI.sections.preconfiguredConnectorForm.flyoutTitle"
+                  values={{ connectorName: initialConnector.name }}
+                />
+                &emsp;
+                <EuiBetaBadge
+                  label="Pre-configured"
+                  tooltipContent={i18n.translate(
+                    'xpack.triggersActionsUI.sections.preconfiguredConnectorForm.tooltipContent',
+                    {
+                      defaultMessage: 'This connector is preconfigured and not allowed to edit',
```

Comment:
This copy doesn't sound right to me.

Perhaps `Preconfigured  connectors cannot be edited` or `This connector is preconfigured and cannot be edited` instead?